### ISSUE TITLE
Bring in historic data container

### ISSUE THREAD

**Issue by jadudm** [GSA-TTS/FAC#1975]
# At a glance

[comment]: # "Begin with a short summary so intent can be understood at a glance."

[comment]: # "In order to: some objective or value to be achieved"
[comment]: # "as a: stakeholder"
[comment]: # "I want: some new feature"

**In order to** do cognizant assignment, test migrations, and meet operational goals
**as a** developer and product owner
**I want** historical data in the system.

# Acceptance Criteria

There is a container at 

containing historic, public data. This data is required for multiple processes in the FAC.

```[tasklist]
### Phase 1
- [x] Update container `wait-and-load` to remove creation of `workbook.generator` user.
- [ ] Incorporate container into TF for `dev`
- [ ] Test deploy of container to `dev`
- [ ] Verify `census_` tables are created
```

Make sure `workbooks_e2e` is healthy.

```[tasklist]
### Phase 2
- [ ] Move creation of `workbook.generator` user to `run.sh` and `.profile` in main repo.
- [ ] Test workbook creation in a local compose
- [ ] Test workbook creation in `dev`
- [ ] Add deploy to `staging`
``` 

It may be that any testing in `dev` will move straight through to `staging`. This probably is fine at this point. Not worth surgery to try and prevent it.

All of this adds some time to our deploy.

The workbook creation code will need some love after we finish updating our ETL process. So, the first steps of just getting the tables created are what we're trying to achieve. Once that is done, we can handle the second set of steps... secondly.

### Shepherd

[comment]: # "@ mention shepherds as we move across the board."

* Design shepherd: 
* Engineering shepherd: 

# Background

[comment]: # "Any helpful contextual notes or links to artifacts/evidence, if needed"

# Security Considerations

Required per [CM-4](https://nvd.nist.gov/800-53/Rev4/control/CM-4).

[comment]: # "Our SSP says 'The team ensures security implications are considered as part of the agile requirements refinement process by including a section in the issue template used as a basis for new work.'"
[comment]: # "Please do not remove this section without care."
[comment]: # "Note any security concerns that might be implicated in the change. 'None' is OK, but we must be explicit here."

There are no particular security concerns to using existing processes for systems deployment. The data in question is arleady public and already disseminated elsewhere. 

# Sketch

We're thinking we'll try implementing A and then changing B.

```[tasklist]
# Tasks
- [ ] [something we'll definitely have to do]
```

---

<details>
  <summary>Process checklist</summary>
  
# Sketch

[comment]: # "Notes or a checklist reflecting our understanding of the selected approach"

- [ ] Design designs all the things
- [ ] Engineering engineers all the things

# Definition of Done

### If not likely to be important in the next quarter...
- [ ] Archived from the board

### Otherwise...

- [ ] Has a clear story statement
- [ ] Design or Engineering accepts that it belongs in their respective backlog

## Design Backlog

- [ ] Has clearly stated/testable acceptance criteria
- [ ] Meets the design Definition of Ready [citation needed]
- [ ] A design shepherd has been identified

## Design In Progress

- [ ] Meets the design Definition of Done [citation needed]

## Design Review Needed

- [ ] Necessary outside review/sign-off was provided

## Design Done

- [ ] Presented in a sprint review
- [ ] Includes screenshots or references to artifacts

### If no engineering is necessary
- [ ] Tagged with the sprint where it was finished
- [ ] Archived

## Engineering Backlog

- [ ] Has clearly stated/testable acceptance criteria
- [ ] Has a sketch or list of tasks
- [ ] Can reasonably be done in a few days (otherwise, split this up!)

## Engineering Available

- [ ] There's capacity in the `In Progress` column
- [ ] An engineering shepherd has been identified

## Engineering In Progress

- [ ] Meets acceptance criteria
- [ ] Meets [QASP conditions](https://derisking-guide.18f.gov/qasp/)

### If there's UI...
- [ ] Screen reader - Listen to the experience with a screen reader extension, ensure the information presented in order
- [ ] Keyboard navigation - Run through acceptance criteria with keyboard tabs, ensure it works. 
- [ ] Text scaling - Adjust viewport to 1280 pixels wide and zoom to 200%, ensure everything renders as expected. Document 400% zoom issues with USWDS if appropriate.

## Engineering Blocked

- [ ] Blocker removed/resolved

## Engineering Review Needed

- [ ] Outside review/sign-off was provided

## Engineering Done

- [ ] Presented in a sprint review
- [ ] Includes screenshots or references to artifacts
- [ ] Tagged with the sprint where it was finished
- [ ] Archived

</details>
